bundles/model/org.eclipse.smarthome.model.script/src/org/eclipse/smarthome/model/script/actions/ScriptExecution.java

```diff
@@ -91,6 +91,10 @@ public static Timer createTimer(AbstractInstant instant, Procedure0 closure) {
             dataMap.put("procedure", closure);
             dataMap.put("timer", timer);
             JobDetail job = newJob(TimerExecutionJob.class).withIdentity(jobKey).usingJobData(dataMap).build();
+            if (TimerImpl.scheduler.checkExists(job.getKey())) {
+                TimerImpl.scheduler.deleteJob(job.getKey());
```

Comment:
Java Object hashes are not what I call "human readable" - but I am afraid you are right - the closures do not come with any proper name or label or toString() that could be potentially used. So let's leave it as it is for now, maybe it can be addressed with #581 then. Thanks.